Scripts/build-mark.py

```
@@ -0,0 +1,135 @@
+    out.paste(bottom, ((w - bottom.width) // 2, top.height + LINE_GAP))
+    return out
+
+
+def tint(src: Image.Image, hexs: str) -> Image.Image:
+    r, g, b = (int(hexs[i : i + 2], 16) / 255 for i in (1, 3, 5))
+    hue, sat, val = colorsys.rgb_to_hsv(r, g, b)
+    out = Image.new("RGBA", src.size, (0, 0, 0, 0))
+    sp, op = src.load(), out.load()
+    for y in range(src.height):
+        for x in range(src.width):
+            l, a = sp[x, y]
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)